src/pages/AboutUs/styles.module.css

```
@@ -0,0 +1,173 @@
+.aboutUs {
+    width: 100vw;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 8%;
+}
+
+.firstStepAboutUS {
+    width: 80vw;
+    display: flex;
+    justify-content: space-between;
+    gap: 80px;
+
+}
+
+.firstStepContentWrapper {
+    width: 60%;
+}
+
+.firstStepContentWrapper h2{
+    font-size: 32px;
+    color: #418671;
+    font-family: 'Poppins', sans-serif;
+    font-weight: bold;
+    margin-bottom: 4%;
+}
+
+.firstStepContentWrapper p, .secondStepContentWrapper p {
+    font-size: 18px;
+    color: #292D34;
+    font-family: 'Poppins', sans-serif;
+    text-align: justify;
+}
+
+.firstStepContentWrapper p:last-child {
+    margin-top: 4%;
+}
+
+.firstStepGraphWrapper {
+    width: 488px;
+    height: 375px;
+    background-image: url('../../assets/img/graph.svg');
+    background-size:contain;
+    background-repeat: no-repeat;
+
+}
+
+.secondStepAboutUs {
+    width: 80vw;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 2%;
+
```

O texto tá muito próximo da lista de contribuidores

```suggestion
    gap: 5vh;
```